src/org/traccar/protocol/Gps103ProtocolDecoder.java

```diff
@@ -189,7 +189,8 @@ protected Object decode(
             getLastLocation(position, dateBuilder.getDate());
 
             position.set(Position.KEY_ODOMETER, parser.nextInt());
-            position.set(Position.KEY_FUEL, parser.next());
+            parser.next();                                                          // instant fuel consumption
```

Comment:
Just to not to be in the middle of the code (up and down). How many would be OK?